layouts/BeltDocsLayout.re

```diff
@@ -94,23 +94,48 @@ module Md = {
     );
 };
 
+module FlavourSwitch = {
+  open BeltDocsFlavour;
+  [@react.component]
+  let make = () => {
+    let flavourContext = useContext();
+
+    <div className="flavour-switch">
+      <span
+        className={flavourContext.flavour == Flavour.Reason ? "active" : ""}
+        onClick={_ => flavourContext.setFlavour(Flavour.Reason)}>
+        {React.string("Reason")}
+      </span>
+      <span
+        className={flavourContext.flavour == Flavour.OCaml ? "active" : ""}
+        onClick={_ => flavourContext.setFlavour(Flavour.OCaml)}>
+        {React.string("OCaml")}
+      </span>
+    </div>;
+  };
+};
+
 module Navigation = {
   let link = "no-underline text-inherit hover:text-white";
   [@react.component]
   let make = () =>
-    <nav className="p-2 flex items-center text-sm bg-bs-purple text-white-80">
-      <Link href="/belt_docs">
-        <a className="flex items-center w-2/3">
-          <img
-            className="h-12"
-            src="https://res.cloudinary.com/dmm9n7v9f/image/upload/v1568788825/Reason%20Association/reasonml.org/bucklescript_bqxwee.svg"
-          />
-          <span
-            className="text-2xl ml-2 font-montserrat text-white-80 hover:text-white">
-            "Belt"->s
-          </span>
-        </a>
-      </Link>
+    <nav
```

Comment:
that's weird, did anything change here?

Reply:
I had to wrap `<Link/>` and `<FlavourSwitch/>` for proper alignment.